### PR TITLE
[UI] Stops experiment list from leaking previous error message

### PR DESCRIPTION
**Pull request by Bobgy**
/cc @chensun 

Fixes https://github.com/kubeflow/pipelines/issues/3296#issuecomment-603066897
/area frontend
/kind bug

### CODE REVIEW

File: frontend/src/pages/ExperimentList.test.tsx [hunk 1]
```diff
@@ -71,17 +72,22 @@ describe('ExperimentList', () => {
     );
   }
 
+  function mockListNExpperiment(n: number = 1) {
```
Comment (chensun) [inline frontend/src/pages/ExperimentList.test.tsx]: nit: `mockListNExpperiment` -> `mockListNExperiment` or `mockListNExpperiments` ?

Comment (Bobgy) [reply frontend/src/pages/ExperimentList.test.tsx]: Thanks! SGTM

**Comment by Bobgy** [comment]
/retest
image build failure

**Pull request merged**